ext/crates/fp/src/vector_inner.rs

```diff
@@ -690,34 +480,37 @@ impl<'a, const P: u32> SliceP<'a, P> {
 impl<'a, const P: u32> SliceP<'a, P> {
     #[inline]
     fn offset(&self) -> usize {
-        let bit_length = bit_length(self.prime());
-        let entries_per_limb = entries_per_limb(self.prime());
+        let bit_length = limb::bit_length::<P>();
+        let entries_per_limb = limb::entries_per_limb_const::<P>();
         (self.start % entries_per_limb) * bit_length
     }
 
     #[inline]
-    fn limb_range(&self) -> (usize, usize) {
+    fn limb_range(&self) -> Range<usize> {
```

Comment:
This change seems to be responsible for a large part of the diff in this file and it seems orthogonal the changes in the rest of the files. Maybe you could do this as a separate PR? (It's certainly a nice improvement.)

Reply:
Thanks for the review! I applied the changes you suggested. Should I "resolve conversation" for your comments above? I'm not sure what the GitHub etiquette is.